receiver/swohostmetricsreceiver/internal/providers/infostat/provider.go

```diff
@@ -74,7 +74,7 @@ func (is *provider) provideInternal(ch chan<- InfoStat) {
 
 	infoStat, err := is.internalExecutor.Getinfo()
 	if err != nil {
-		zap.L().Error("InfoStat command execution failed", zap.Error(err))
+		is.logger.Error("InfoStat command execution failed", zap.Error(err))
```

Comment:
```suggestion
		is.logger.Error("infostat command execution failed", zap.Error(err))
```
this is infostat https://pkg.go.dev/github.com/shirou/gopsutil/host#InfoStat, it is upper-cased but I think we can have it in lower case.